docs/data/material/guides/server-rendering/server-rendering.md

```diff
@@ -218,6 +218,7 @@ We host different reference implementations which you can find in the [GitHub re
 - [The reference implementation of this tutorial](https://github.com/mui/material-ui/tree/HEAD/examples/ssr)
 - [Gatsby](https://github.com/mui/material-ui/tree/HEAD/examples/gatsby)
 - [Next.js](https://github.com/mui/material-ui/tree/HEAD/examples/nextjs) ([TypeScript version](https://github.com/mui/material-ui/tree/HEAD/examples/nextjs-with-typescript))
+- [Remix.run](https://github.com/mui/material-ui/tree/HEAD/examples/remix-with-typescript)
```

Comment:
I think that we should point to master as once we start to work on MUI v6, there will be breaking changes, but we will make next the head, which will break all the links on https://mui.com/.

```suggestion
<!-- #default-branch-switch -->

- [The reference implementation of this tutorial](https://github.com/mui/material-ui/tree/master/examples/ssr)
- [Gatsby](https://github.com/mui/material-ui/tree/master/examples/gatsby)
- [Next.js](https://github.com/mui/material-ui/tree/master/examples/nextjs) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/nextjs-with-typescript))
- [Remix.run](https://github.com/mui/material-ui/tree/master/examples/remix-with-typescript)
```

Reply:
Resolved. Also updated the Chinese and Portuguese docs.